send/pithia_portal.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+import os
+import sys
+import subprocess
+import send_lib
+import requests
+
+SERVICE_NAME = "pithia_portal"
+os.environ["SERVICE_NAME"] = SERVICE_NAME
+
+facility = sys.argv[1]
+destination = sys.argv[2]
+destination_type = sys.argv[3]
+auth_token = None
+
+auth = send_lib.get_auth_credentials(SERVICE_NAME, destination)
+tokenUrl = send_lib.get_custom_config_properties(SERVICE_NAME, destination, ["tokenUrl"])[0]
+data = {
+	"username": auth[0],
+	"password": auth[1]
+}
+response = requests.post(tokenUrl, data)
+
+if response.status_code == 200:
+	response = response.json()
+	if "token" not in response:
+		send_lib.die_with_error("Token missing in response from token server")
+	auth_token = response["token"]
+else:
+	send_lib.die_with_error("Response from token server ended with error code: " + str(response.status_code)
+							+ ", content: " + response.text)
+
+# TODO update with function `main` function call when merged
```

Comment:
As mentioned in TODO, since #822 is merged all code below can be replaced with:
```python
opts = ['-H', 'Authorization: Bearer '+auth_token, '--cookie', 'sessionid='+resp_cookies['sessionid'] ]

generic_sender.send(service_name, facility, destination, destination_type, opts)
```